bip-XXXX.mediawiki

```diff
@@ -0,0 +1,449 @@
+<pre>
+  BIP: ???
+  Layer: Consensus (soft / hard fork)
+  Title: Logarithm of transaction fee limits block size. Game theoretic scaling on chain.
+  Author: Simon Cavegn <random@gmx.ch>
+  Comments-Summary: None
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-XXXX
+  Status: Draft
+  Type: Standards Track
+  Created: 2019-03-28
+</pre>
+
+==Abstract==
+
+Logarithm of transaction fee limits block size.
+
+If a sender is willing to pay a large fee, then a transaction may be allowed in a block even if blocksize for lower fee-transactions is full. Yet the blocksize is limited harshly by a formula.
```

Comment:
The abstract is concise, but does not sufficiently describe either the solution or the technical issue being addressed.